### PR TITLE
fix(deps): Update kotlin to 1.9

### PR DESCRIPTION
**Pull request by sdsantos**
Addresses https://github.com/relaycorp/relayverse/issues/47

### CODE REVIEW

**Comment by gnarea** [comment]
@sdsantos, were all `.kt` files changed by ktlint? Just wondering if I need to write anything in there

**Comment by sdsantos** [comment]
> @sdsantos, were all `.kt` files changed by ktlint? Just wondering if I need to write anything in there

Yup, most were whitespace changes made by ktlint. You can hide those changes from the "Files changes" tab to save your the trouble:

Screenshot 2023-11-28 at 16 13 27

**Review by gnarea — DISMISSED** [review]
Cool, thanks!

File: app/build.gradle [hunk 1]
```diff
@@ -61,9 +61,6 @@ android {
     }
 
     buildTypes {
-        debug {
-            testCoverageEnabled true
-        }
```
Comment (gnarea) [inline app/build.gradle:66]: Isn't this disabling test coverage? Why is this needed to fix the gRPC tests?

Comment (sdsantos) [reply app/build.gradle]: Test coverage still runs without it, but I believe it will struggle to merge coverage from unit and instrumented tests. We were having some warnings related to it, but I just checked and they don't affect the tests passing, so I restored it.

**Review by gnarea — APPROVED** [review]
